cumulusci/cli/cci.py

```diff
@@ -811,6 +811,19 @@ def org_connect(config, org_name, sandbox, login_url, default, global_org):
     oauth_dict = oauth_capture()
     org_config = OrgConfig(oauth_dict, org_name)
     org_config.load_userinfo()
+    org_config._load_orginfo()
+    if (
+        org_config.organization_sobject
+        and "TrialExpirationDate" in org_config.organization_sobject.keys()
+    ):
+        if org_config.organization_sobject["TrialExpirationDate"] is None:
+            org_config.config["expires"] = "Persistent"
+        else:
+            org_config.config["expires"] = parse_api_datetime(
+                org_config.organization_sobject["TrialExpirationDate"]
+            ).strftime("%m/%d/%Y")
```

Comment:
I want to store the actual datetime object, not a string formatted from it. Then do the formatting [where it's used in output](https://github.com/SFDO-Tooling/CumulusCI/pull/1320/files#diff-34b79c3fe3ee83def757b70a69cb5770R974). That way it's not a problem if we decide we want to show it in a different format, or do some date arithmetic based on it.

Also I would use dt.date().isoformat() to format the date as YYYY-MM-DD since that's easier to understand in different locales (it's primarily just the US that uses mm/dd/YYYY)